easypost-api-client.iml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file for?

Reply:
It's a module settings file created by IntelliJ IDEA, I will add it to the **gitingore** file